tests/test_rclone.py

```
@@ -0,0 +1,102 @@
+@pytest.fixture
+def _mock_graphql() -> AsyncMock:
+    with patch("unraid_mcp.tools.rclone.make_graphql_request", new_callable=AsyncMock) as mock:
+        yield mock
+
+
+def _make_tool():
+    return make_tool_fn("unraid_mcp.tools.rclone", "register_rclone_tool", "unraid_rclone")
+
+
+class TestRcloneValidation:
+    async def test_delete_requires_confirm(self, _mock_graphql: AsyncMock) -> None:
```

Ruff (0.15.0)

[warning] Fixture `_mock_graphql` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)